ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Impl.hs

```diff
@@ -281,9 +281,12 @@ implAddTxs mpEnv accum txs = assert (all txInvariant txs) $ do
 
     let ValidationResult { vrNewValid = accepted } = vr
 
-    traceBatch TraceMempoolRemoveTxs   mempoolSize (map fst removed)
-    traceBatch TraceMempoolAddTxs      mempoolSize accepted
-    traceBatch TraceMempoolRejectedTxs mempoolSize rejected
+    -- We get the time for the measurement of the txs added into the mempool.
+    time <- getMonotonicTime
+
+    traceBatch TraceMempoolRemoveTxs   mempoolSize (map fst removed) time
+    traceBatch TraceMempoolAddTxs      mempoolSize accepted time
+    traceBatch TraceMempoolRejectedTxs mempoolSize rejected time
```

Comment:
Can you align on `time`? We like alignment, as it helps us spot the differences between similar looking lines.